### PR TITLE
Remove elasticsearch, cyjax from py3-tools

### PR DESCRIPTION
**Pull request by ilaner**
## Status
Ready/In Progress/In Hold (Reason for hold)

## Related Content Pull Request
Related PR: link to the PR at demisto/content

## Related Issues
Related: link to the issue

## Description
Those libraries require `urllib3 1`. Which is incompatible to our native image

### CODE REVIEW

**Comment by JudahSchwartz** [comment]
please remove pipfile in the pr

**Comment by GuyAfik** [comment]
@ilaner please do the same for `py3-tools-ubi` which is what `py3-native` uses.

**Review by JudahSchwartz — APPROVED** [review]

File: docker/py3-tools-ubi/pyproject.toml [hunk 1]
```diff
@@ -41,16 +41,13 @@ gql = "*"
...
 pymongo = "*"
-cyjax-cti = "*"
 pyzabbix = "*"
 jmespath = "*"
 tldextract = "*"
 stix2 = "*"
 taxii2-client = "*"
 xlrd = "==1.2.0"
-elasticsearch = "*"
 opensearch-py = "*"
```
Comment (GuyAfik) [inline docker/py3-tools-ubi/pyproject.toml:50]:
```suggestion
```

we might not need this one too, as its also part of the `elasticsearch` image

**Review by tkatzir — APPROVED** [review]
Great!

**Review by GuyAfik — APPROVED** [review]
👑

**Pull request merged**